docs/ja/changelog.md

```
@@ -1094,5 +1094,5 @@ outline: deep
 - ReDoS（正規表現サービス拒否）防止
 - 全 API エンドポイントの入力検証
 
-[unreleased]: https://github.com/geolonia/geonicdb/compare/v0.1.0...HEAD
-[0.1.0]: https://github.com/geolonia/geonicdb/releases/tag/v0.1.0
\ No newline at end of file
+[unreleased]: #unreleased
+[0.1.0]: #0-1-0
```

markdownlint-cli2 (0.22.1)

[warning] Link fragments should be valid

(MD051, link-fragments)